### PR TITLE
Fix typo in virtctl imageupload usage example

### PR DESCRIPTION
**Pull request by aglitke**
Signed-off-by: Adam Litke <alitke@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by mfranczy** [comment]
@aglitke could you backport this as well to release-0.9 branch after PR is merged? I think we have to rebuild virtctl rpm.

File: pkg/virtctl/imageupload/imageupload.go [hunk 1]
```diff
@@ -116,7 +116,7 @@ func NewImageUploadCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 
 func usage() string {
 	usage := `  # Upload a local disk image to a newly created PersistentVolumeClaim:
-	virtctl image-upload --upload-proxy-url=https://cdi-uploadproxy.mycluster.com --pvc-name=upload-pvc --pvc-size=10Gi --image-path=/images/fedora28.qcow2`
+	virtctl image-upload --uploadproxy-url=https://cdi-uploadproxy.mycluster.com --pvc-name=upload-pvc --pvc-size=10Gi --image-path=/images/fedora28.qcow2`
```
Comment (mhenriks) [inline pkg/virtctl/imageupload/imageupload.go]: This really a typo?  The container/service is `cdi-uploadproxy`.

Comment (aglitke) [reply pkg/virtctl/imageupload/imageupload.go]: It doesn't match the actual parameter which is `--uploadproxy-url`

Comment (mhenriks) [reply pkg/virtctl/imageupload/imageupload.go]: Ha I was seeing reverse.  Sorry.  My brain is clearly turned off and in vacation mode.  No more githubbing until the 26th.

**Pull request merged**